common/schemas.py

```diff
@@ -1,4 +1,46 @@
+from dataclasses import dataclass
+
 # Numbers represent the prefix of the id of the cards.
 # i.e. 26000037 is a Inferno Dragon, 27000007 is Elixir Collector, 28000000 is fireball
 CARD_TYPE_ID_PREFIX = {'26': 'troop', '27': 'building', '28': 'spell'}
 MAX_CARD_LEVEL = 14
+
+@dataclass
+class PlayerActivity:
+    '''
+    A player has a  "WarActiveWeek" for a specific week IF:n
```

Comment:
/supernit I know I wrote this lol. But pls fix, should be `if:` instead of `IF:n`